content/docs/list-components/SingleDropdownList.md

```diff
@@ -119,6 +119,12 @@ or
 - **showLoadMore** `Boolean` [optional]  
     defaults to `false` and works only with elasticsearch >= 6 since it uses [composite aggregations](https://www.elastic.co/guide/en/elasticsearch/reference/current/search-aggregations-bucket-composite-aggregation.html). This adds a "Load More" button to load the aggs on demand combined with the `size` prop. Composite aggregations are in beta and this is an experimental API which might change in a future release.
 
+## Demo
+
+<br />
+
+<iframe src="https://codesandbox.io/s/m9j1k1rq9p" style="width:100%; height:500px; border:0; border-radius: 4px; overflow:hidden;" sandbox="allow-modals allow-forms allow-popups allow-scripts allow-same-origin"></iframe>
+
```

Comment:
@divyanshu-rawat Please add SelectedFilters in all codesanbox demos.
And keep the same layout in full screen mode as we have in storybook demos( in this example dropdown list covers the results ).